api/server/sdk/server.go

```diff
@@ -583,6 +592,9 @@ func (s *sdkGrpcServer) Start() error {
 		if s.jobServer != nil {
 			api.RegisterOpenStorageJobServer(grpcServer, s.jobServer)
 		}
+		if s.scheduleServer != nil {
```

Comment:
we dont need this check, given that the scheduleServer always gets initialized on line 474?